.gitignore

```diff
@@ -2,6 +2,8 @@
 # will have compiled files and executables
 /target/
 
+wasm/target
```

Comment:
I think it would be more consistent to have `.gitignore` inside `wasm` directory (it is done like this with Python bindings) + the main crate will not know about the dependent crate